Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -78,17 +75,47 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
 
 - (BOOL)answerCall:(OCTCall *)call enableAudio:(BOOL)enableAudio enableVideo:(BOOL)enableVideo error:(NSError **)error
 {
-    return NO;
+    OCTToxAVAudioBitRate audioBitRate = (enableAudio) ? kDefaultAudioBitRate : kOCTToxAVAudioBitRateDisable;
+    OCTToxAVVideoBitRate videoBitRate = (enableVideo) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+
+    if (call.chat.friends.count == 1) {
+
+        OCTFriend *friend = call.chat.friends.firstObject;
+
+        BOOL status = [self.toxAV answerIncomingCallFromFriend:friend.friendNumber audioBitRate:audioBitRate videoBitRate:videoBitRate error:error];
+
+        if (status) {
+            self.audioEngine.friendNumber = friend.friendNumber;
+            [self.audioEngine startAudioFlow:error];
+            [self.calls updateCall:call updateBlock:^(OCTCall *call) {
```

Comment:
This naming hides old `call` variable, which may lead to errors in future. You can rename second `call` (that is used in block) to `callToUpdate` or something.

Same for other blocks here.